### DISCUSSION TITLE
Make whole datasheet read-only

### DISCUSSION THREAD

**Discussion by alenocera** [nick-keller/react-datasheet-grid#191] [Feature requests]
Hi, 
after reading the documentation I'm able to discover that there are only the columns/cells disable possibility.
There is a way to have a props named `readonly` in order to make the whole **datasheet** in _read-only_ mode (just possible copy data in this way)? 

Thanks a lot!

**Comment by nick-keller** [comment]
That is true, good point. A workaround would be to set [`disabled`](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#disabled) to true for each column. You should also set [`lockRows`](https://react-datasheet-grid.netlify.app/docs/api-reference/props#lockrows) to true on the grid.
You could also [change some CSS](https://react-datasheet-grid.netlify.app/docs/styling#generator) variables to make the UI look "normal"
